### ISSUE TITLE
Heading/sectioning issues

### ISSUE THREAD

**Issue by greatislander** [pressbooks/pressbooks-network-catalog#58]
Right now the headings on the page are as follows:

H1: Catalog
H2: 11 Results

I think the following would be an improvement in terms of meaningfully titling the different sections:

H1: Catalog
H2 (screen reader only): Filters (this is currently an aside but I don't think it should be; `<aside>` elements are only supposed to be "indirectly related to the main content")
H2 (screen reader only): Search, pagination, and sorting
H2 (screen reader only): Books

You might want to also consider making the filters and the search/pagination/sorting controls into sections with an aria-label instead of a screen reader-only H2 (see here for some context: https://css-tricks.com/the-deal-with-the-section-element/)

Key takeaway:

> When a section element has an accessible name it becomes a navigable landmark element, so a screen reader user can use their screen reader’s shortcut key for navigating from one to the next – just like they can do with headings.
